test/e2e/config/packet-dev.yaml

```diff
@@ -57,7 +57,7 @@ providers:
     - old: "--enable-leader-election"
       new: "--enable-leader-election=false"
   files:
-  - sourcePath: "../../../templates/metadata-template.yaml"
+  - sourcePath: "../../metadata.yaml"
```

Comment:
When you run the e2e tests how can we be sure that the metadata.yaml file exists? It is not part of the repo for what I can see